### ISSUE TITLE
Check ligatures are consistent

### ISSUE THREAD

**Issue by davelab6** [fonttools/fontspector#455]
## Observed behaviour

https://twitter.com/mjdye/status/1223762539768467456?s=19

> So, I’m playing around with @ebensorkin 's Gelasio on google fonts and have noticed ligatures are weirdly inconsistent. See screenshot. Is that a mistake or is that intentional? https://t.co/UaXivIpZ1z

![IMG_20200201_200959](https://user-images.githubusercontent.com/261579/73601408-05b8d480-452f-11ea-8923-0015d97cb2e4.jpg)

## Expected behaviour

Ligatures should be technically checked for existing, and visually checked for similarities. The latter might be more of an exq thing

**Comment by felipesanches** [comment]
could you please clarify "exq thing"?

**Comment by felipesanches** [comment]
Are we discussing here a proposal of a check specific to `fi-fl-ffi-ffl` ligatures?

Or are we talking bout a broader issue that involves a larger set of potential ligatures?

**Comment by felipesanches** [comment]
also...

* **"checked for existing"** sounds like something easily scriptable as long as we have a list of pairs to check against
* **"visually checked for similarities"** sounds like a task for actual human type-designers to do so it might be dashboard-level.

**Comment by davelab6** [comment]
Eqx, by sorkin type

**Comment by EbenSorkin** [comment]
Yeah this could be an EQX pass fail question or possibly 2+ basic questions which build to a family consistency question. I'll have a look at this post Merriweather and revise the source. For some reason I don't recall if I made Gelasio into a VF. If I somehow didn't I'll do that as well.

**Comment by felipesanches** [comment]
All of Font Bakery's `"new check proposal"` issues are being either closed or migrated to the [Fontspector issue tracker](https://github.com/fonttools/fontspector/issues), as there's no intention to implement new checks on the Font Bakery codebase anymore.

If you believe this proposal is still valid, please post a message here asking for it to be migrated. Otherwise, on **July 20, 2025** (30 days from now) it will be closed due to inactivity.

**Comment by felipesanches** [comment]
The fontspector issue tracker does not currently use priority/importance labels, but when it was hosted at the fontbakery issue tracker:

* @felipesanches applied "P2 Important" on Feb 11, 2020

**Comment by simoncozens** [comment]
I don't have enough actionable information here to implement this, and there's been no updates since it was deprecated.
